bus/routes.go

```diff
@@ -2320,6 +2320,13 @@ func (b *Bus) contractsFormHandler(jc jape.Context) {
 			return
 		}
 
+		// cap v1 formations to the v2 require height since the host won't allow
```

Comment:
It's probably more developer friendly to return a meaningful error rather than just change the requested end height. Every other API would likely 400 here, not sure we should care a lot though, but still